db/seeds.rb

```diff
@@ -5,3 +5,6 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+user1 = User.find(1)
+restaurant1 = Restaurant.find(642095277)
```

Comment:
Just something to keep in mind when you write your README instructions for someone to setup a version of this repo on their own machine, you may not want to include instructions to run the seed file, this seed file is assuming a User will already be created and be in the database